integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/AwsSqsSteps.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.awaitility.Awaitility;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.smartevents.integration.tests.resources.AwsSqsResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+public class AwsSqsSteps {
+
+    private TestContext context;
+
+    public AwsSqsSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @And("^create an SQS queue on AWS called \"([^\"]*)\" within (\\d+) (?:minute|minutes)$")
+    public void createQueue(String queueName, int timeoutMinutes) {
+        Awaitility.await()
+                .atMost(Duration.ofMinutes(timeoutMinutes))
+                .untilAsserted(() -> AwsSqsResource.createQueue(queueName));
```

Comment:
Why do you use awaitility here? Does it mean that `AwsSqsResource.createQueue(queueName))` can fail or times out?

Reply:
my bad, thanks for spotting

Reply:
looks good now